pymtl/tools/translation/verilog_behavioral.py

```diff
@@ -541,7 +541,7 @@ def visit_Call(self, node):
       sig_name   = self.visit( node.args[0] )
       sig_nbits  = node.args[0]._object.nbits
       ext_nbits  = self.visit( node.args[1] )
-      return '{{ {{ {3}-{1} {{ {0}[{2}] }} }}, {0}[{2}:0] }}' \
+      return '{{ {{ {3}-{1} {{ {0}[{2}] }} }}, {0} }}' \
```

Comment:
This extra slice here is unnecessary as far as I can tell, and causes it to generate verilog that, while accepted by verilator, is not accepted by vivado.